great_expectations/render/renderer_configuration.py

```diff
@@ -299,9 +300,10 @@ def _validate_and_set_renderer_attrs(cls, values: dict) -> dict:
                     if "_params" in values and values["_params"]
                     else renderer_params_args
                 )
-        else:
+        elif "configuration" in values and values["configuration"] is not None:
```

Comment:
Does `values` has some sort of guaranteed structure? Does a `TypedDict` have value here? (not high priority if this needs to get in asap)

Reply:
No, it depends on what gets passed to the class. `pydantic` will not add the key to the `dict` if it fails the type validation (which is separate from this validator).